sources/packages/forms/src/form-definitions/sfaa2024-25.json

```diff
@@ -31583,7 +31582,7 @@
                           "value": ""
                         }
                       ],
-                      "content": "<h4 class=\"category-header-medium-small\">Documents Required</h4>\n<ul>\n  <li>A letter describing your situation to indicate why an exception should be made (including why you cannot move closer to your school)</li>\n<li>A copy of your vehicle insurance</li>\n<li>All relevant supporting documentation (work schedule, clinical or practicum information, etc.)</li>\n</ul>",
+                      "content": "<h4 class=\"category-header-medium-small\">Documents Required</h4>\n<ul>\n  <li>Letter describing your situation to indicate why an exception should be made (including why you cannot move closer to your school)</li>\n<li>Copy of your vehicle insurance</li>\n<li>All relevant supporting documentation (work schedule, clinical or practicum information, etc.)</li>\n</ul>",
```

Comment:
Can you please let me know where this change is requested in the Story?

Reply:
As discussed, an accidental persist from old form versions. Updated.